data-processing/pulsar-flink/1.13/pulsar-flink.md

```diff
@@ -8,7 +8,7 @@ source: "https://github.com/streamnative/pulsar-flink/tree/release-1.13"
 license: Apache License 2.0
 license_link: "https://www.apache.org/licenses/LICENSE-2.0"
 tags: ["Pulsar", "Flink", "Connector"]
-alias: Pulsar Flink Connector
+alias: Pulsar Flink Connector (Legacy)
```

Comment:
I prefer using "Deprecated" here, meaning that we do not plan to maintain it in the future.

Reply:
@teachwrite could you please give some suggestions? Thanks.

Reply:
Depreciated is typically used when we want people to use a newer or different version, so that word would seem appropriate in this situation (there's a new version we want people to move to).